### PR TITLE
docs(core): update readme in English and Chinese

### PR DESCRIPTION
**Pull request by pswu11**
- update intro
- improve get started section and structure
- include client libraries
- update bench results image 
- remove link to GitHub Discussions

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `build`, `ci`, `chore`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2a2d562cf551750204585a73236c108b4473e288
</p>

File: README.md [hunk 1]
```diff
@@ -74,16 +75,33 @@ methods.
...
+- [C/C++](https://github.com/questdb/c-questdb-client)
+- [Go](https://pkg.go.dev/github.com/questdb/go-questdb-client)
+- [Java](https://questdb.io/docs/reference/clients/java_ilp/)
+- [NodeJS](https://questdb.github.io/nodejs-questdb-client)
+- [Python](https://py-questdb-client.readthedocs.io/en/latest/)
+- [Rust](https://docs.rs/crate/questdb-rs/latest)
 
 ## How QuestDB compares to other open source TSDBs
 
+[This article](https://questdb.io/blog/2021/07/05/comparing-questdb-timescaledb-influxdb/)
```
Comment (amyshwang) [inline README.md:101]: Any reason we don't use this one too?

**Review by amyshwang — APPROVED** [review]
left one question - rest looks good.

File: README.md [hunk 2]
```diff
@@ -19,19 +19,18 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
...
+QuestDB is a high-performance, open-source time-series database for applications
+where timely data ingestion and querying is critical. It is widely used with
+financial market data, application metrics, sensor data, realtime analytics and
+dashboards, and infrastructure monitoring.
+
+QuestDB uses ANSI SQL and benefits from time-series SQL semantics. These SQL
+semantics make it simple to correlate data from multiple sources using
+relational and time-series joins. QuestDB achieves high performance from a
+column-oriented storage model, parallelized vector execution, SIMD instructions,
+and various low-latency techniques. The entire codebase was built from the
```
Comment (tris0laris) [inline README.md:31]: The entire codebase was built  >> is built

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
@pswu11 there are readme's in other languages too, they need to be maintained together

@amyshwang @tris0laris do not leave PR in approved state if you're requesting changes.

File: README.md (hunk 1 above)
Comment (pswu11) [reply README.md:101]: not sure I understand your question?

**Comment by pswu11** [comment]
@amyshwang I updated the Chinese versions too, please help to review, thanks!

**Comment by pswu11** [comment]
Hi @bluestreak01 sure, for specific languages like Arabic we'll need help from contributors, and for Italian perhaps @tris0laris could help in a separate PR? Spanish one will be in sync with this one.

**Comment by bluestreak01** [comment]
Alex and/or I will edit Ukrainian test over the weekend

File: README.md [hunk 3]
```diff
@@ -19,19 +19,21 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
...
+QuestDB is well suited for financial market data, application metrics, sensor
+data, realtime analytics, dashboards, and infrastructure monitoring.
+
+QuestDB uses ANSI SQL and benefits from time-series SQL semantics. These SQL
+semantics make it simple to correlate data from multiple sources using
+relational and time-series joins. QuestDB achieves high performance from a
+column-oriented storage model, parallelized vector execution, SIMD instructions,
+and various low-latency techniques. The entire codebase is built from the ground
+up in Java and C++, with no dependencies, and is 100% free from garbage
+collection.
```
Comment (argshook) [inline README.md:36]: trying to simplify these sentences, reduce repetition.

```suggestion
QuestDB uses ANSI SQL and benefits from time-series SQL semantics. They
make it easy to correlate data from multiple sources using relational
and time-series joins.

High performance is achieved from a column-oriented storage model,
parallelized vector execution, SIMD instructions and various low-latency
techniques. The entire codebase is built from scratch in Java and C++,
with no dependencies, and is 100% free from garbage collection.
```

Comment (pswu11) [reply README.md:36]: Thanks! for the second paragraph tho I'd still avoid passive voice (which is generally not recommended in technical writing.)

Comment (argshook) [reply README.md:36]: Id like to avoid every paragraph starting with `QuestDB`, looks very monotonic, that's why choice different phrase

Comment (pswu11) [reply README.md:36]: @argshook  brainstorming, how about

> We optimize the performance using column-oriented storage model, parallelized vector execution, SIMD instructions, and various low-latency techniques.

or 

> We achieve high performance using column-oriented storage model, parallelized vector execution, SIMD instructions, and various low-latency techniques.

Comment (argshook) [reply README.md:36]: Maybe the second one?

I updated my code suggestion with that one, if you'd choose to use it

**Review by argshook — DISMISSED** [review]
I'm only able to assess english part, which looks good to me!

File: README.md [hunk 4]
```diff
@@ -19,19 +19,20 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
...
+QuestDB is an open-source time-series database for high throughput ingestion and
+fast SQL queries with operational simplicity. It supports schema-agnostic
+ingestion using the InfluxDB line protocol, PostgreSQL wire protocol, and a REST
+API for bulk imports and exports.
+
+QuestDB is well suited for financial market data, application metrics, sensor
+data, real-time analytics, dashboards, and infrastructure monitoring.
+
+QuestDB uses ANSI SQL and benefits from time-series SQL semantics. It is simple
+to correlate data from multiple sources using relational and time-series joins.
```
Comment (tris0laris) [inline README.md:31]: QuestDB uses ANSI SQL and benefits from time-series SQL semantics. It is simple
to correlate data from multiple sources using relational and time-series joins.

I would change to

QuestDB implements ANSI SQL with native time-series SQL semantics. These semantics 
make it easy to correlate data from multiple sources using relational and time-series joins.

File: README.md [hunk 5]
```diff
@@ -19,19 +19,20 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
...
+We achieve high performance from a column-oriented storage model,
+parallelized vector execution, SIMD instructions, and various low-latency
```
Comment (tris0laris) [inline README.md:33]: We achieve high performance from a column-oriented storage model,
parallelized vector execution, SIMD instructions, and various low-latency
techniques.

I would change to

QuestDB's architecture is focused on performance with a column-oriented storage model,
parallelized vector execution, SIMD instructions, and low-latency
techniques.

File: i18n/README.zh-cn.md [hunk 6]
```diff
@@ -32,11 +32,15 @@
...
-Line 协议以提供不受数据库模式影响的高吞吐数据获取能力，并提供用于查询、批量导入
-和导出的 REST API。QuestDB 使用了 ANSI SQL ，其包含时间导向的原生扩展语言功能。
-这些扩展能更简单的连接（JOIN）多个来源的关联数据以及时间序列数据。QuestDB 通过列导向的存储模型、大规模并行的矢量执行、SIMD 指令和各种低延迟技术实现了高性能。整个代码库是用 Java 和 C++从头开始构建的，没有任何外部依赖，并且 100% 不受垃圾回收的影响。
+QuestDB 是一个开源的時序数据库，支持高吞吐数据获取和快速 SQL 查询并且操作简单。
+应用场景包括金融市场数据、传感器数据、实时分析、数据仪表板和基础设施监控。它兼容
+InfluxDB 行协议以提供不受数据库模式影响的高吞吐数据获取能力，也兼容 PostgreSQL
+的线路协议，並提供 REST API 以便于批量导入和导出。
+
+QuestDB 使用了 ANSI SQL ，並包含时间导向的 SQL 语句。这些 SQL 语句能更简单的连接
+（JOIN）多个来源的关联数据以及时间序列数据。QuestDB 通过列导向的存储模型、大规模
```
Comment (amyshwang) [inline i18n/README.zh-cn.md:41]:
```suggestion
（JOIN）不同来源的关联数据以及时间序列数据。QuestDB 通过列导向的存储模型、大规模
```

File: i18n/README.zh-hk.md [hunk 7]
```diff
@@ -32,9 +32,15 @@
 
 # QuestDB
 
-QuestDB 是一個高性能的開源 SQL 數據庫，適用於金融服務、物聯網、機器學習、DevOps 和可觀測性領域的應用場景。它兼容 PostgreSQL 的 wire 協議，也兼容 InfluxDB Line 協議以獲取不受模式影響的高吞吐量數據，以及提供用於查詢、批量導入和導出的 REST API。
+QuestDB 是一個開源的 SQL 時序數據庫，支持高吞吐量數據和快速 SQL 查詢並且操作簡單
+。其中應用範圍包括金融市場數據、感測器數據、實時數據、儀表板和基礎設施監控。它兼
```
Comment (amyshwang) [inline i18n/README.zh-hk.md:36]:
```suggestion
。其應用範圍包括金融市場數據、感測器數據、實時數據、儀表板和基礎設施監控。它兼
```

File: i18n/README.zh-hk.md [hunk 8]
```diff
@@ -32,9 +32,15 @@
...
+容 InfluxDB 行協議以提供不受模式影響的高吞吐量數據，也兼容 PostgreSQL 線路協議，並
+提供 REST API 以便於批量導入和導出。
 
-QuestDB 使用了包含時間導向的原生擴展語言功能的 ANSI SQL。這些擴展能更簡單的連接（JOIN）多個來源的關聯數據以及時間序列數據。QuestDB 通過列導向的存儲模型、大規模並行的矢量執行、SIMD 指令和各種低延遲技術實現了高性能。整個代碼庫是用 Java 和 C++從頭開始構建的，沒有任何外部依賴，並且 100% 不受垃圾回收的影響。
+QuestDB 使用了 ANSI SQL，並包含了時間導向的 SQL 語句。這些 SQL 語句能更簡單的連
```
Comment (amyshwang) [inline i18n/README.zh-hk.md:40]:
```suggestion
QuestDB 使用ANSI SQL，並包含了時間導向的 SQL 語句。這些 SQL 語句能更簡單的連
```

**Review by amyshwang — CHANGES_REQUESTED** [review]
Note that my suggestion is in traditional Chinese...

**Review by tris0laris — APPROVED** [review]

**Review by bluestreak01 — APPROVED** [review]

**Review by amyshwang — APPROVED** [review]

**Pull request merged**